commands/issue.go

```diff
@@ -157,12 +182,20 @@ func listIssues(cmd *Command, args *Args) {
 	if args.Noop {
 		ui.Printf("Would request list of issues for %s\n", project)
 	} else {
-		filters := map[string]interface{}{}
-		if cmd.FlagPassed("state") {
-			filters["state"] = flagIssueState
+		flagFilters := map[string]string{
+			"state":     flagIssueState,
+			"assignee":  flagIssueAssignee,
+			"milestone": flagIssueMilestoneFilter,
+			"creator":   flagIssueCreator,
+			"mentioned": flagIssueMentioned,
+      "labels":    flagIssueLabelsFilter,
+			"since":     flagIssueSince,
```

Comment:
I still think we need to process `flagIssueSince` a bit before passing it on to the API. The reason is timezones.

A person in New Zealand might use `2016-08-19` to get all isues on or after Friday. However, when the API interprets that date, most likely in the PST timezone, the results returned from start of Friday will be very different, ignoring most issues opened on Friday in New Zealand.

We have to assume that when a person doesn't specify a time or timezone offset, `2016-08-19` means “the start of the 19th of August for me in my local timezone“. So, we should parse the date in the local timezone, have it default to `00:00:00` for time, and serialize it into ISO 8601 format when passing to API so the time is absolutely precise and unambiguous.